codegen/cmd/injection-gen/generators/packages.go

```diff
@@ -193,13 +193,13 @@ func extractCommentTags(t *types.Type) map[string]map[string]string {
 	return ExtractCommentTags("+", comments)
 }
 
-func extractReconcilerClassTag(tags map[string]map[string]string) (classname string, has bool) {
-	vals, has := tags["genreconciler"]
-	if !has {
-		return
+func extractReconcilerClassTag(tags map[string]map[string]string) (string, bool) {
+	vals, ok := tags["genreconciler"]
+	if !ok {
+		return "", false
 	}
-	classname, _ = vals["class"]
-	return
+	classname, has := vals["class"]
```

Comment:
is `return vals["class"]` the same here?

Reply:
No, I tried that first :(

Go special cases subscript on maps to have one or two values, but it isn't smart enough to use the latter here.